### PR TITLE
HW2_A.Bondar

### PR DESCRIPTION
**Pull request by sashulis4e**

### CODE REVIEW

File: A_Bondar/HW2/HW1.1.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+M = int(input('Введите рубли'))
+N = int(input('Введите копейки'))
+L = int(input('Введите количество товара'))
+print((M + N / 100) * L)
```
Comment (belovmd) [inline A_Bondar/HW2/HW1.1.py:4]: with a price 1ruble 50 kopeks and 3 pieces your algorithm will give (50+1)*3/100=1.53

File: A_Bondar/HW2/HW1.fibona44i.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+n = int(input('Enter number'))
```
Comment (belovmd) [inline A_Bondar/HW2/HW1.fibona44i.py:1]: we need only n'th number and don't need to print all others

**Review by belovmd — CHANGES_REQUESTED** [review]
1. Please add other HW tasks. 
2. Please add task description to docstrings
3. Please address these issues
4. Please name PR according to our guidelines

File: A_Bondar/HW2/HW1.1.py (hunk 1 above)
Comment (sashulis4e) [reply A_Bondar/HW2/HW1.1.py:4]: I'm afraid, you are wrong. Please check the sequence of mathematical actions.
1. 50/100=0,5
2. 1+0,5=1,5
3. 1,5*3=4,5
Thanks.

Comment (belovmd) [reply A_Bondar/HW2/HW1.1.py:4]: sure

**Review by belovmd — COMMENTED** [review]
5
overdue, redundant tasks from 4th homework, insufficient tasks from hw2, failed codestyle

**Pull request closed without merging**